scripts/goldpolish-target.py

```diff
@@ -129,6 +129,12 @@ def get_mapping_info(minimap2, ntLink):
         raise ValueError("Unknown mapping tool")
     return (mapper, s_goldpolish)
 
+def split_output_file_name(output_file):
+    """Removes .fa from end of output file to create a prefix"""
+    if output_file.endswith('.fa'):
```

Comment:
To be safe, I think we should also handle other conventional fasta suffices? (.fasta, .fna)

Reply:
Good catch!